OneCommunityGlobal/HighestGoodNetworkApp
# Vishala Check Types Modal

# Description
This PR implements a Modal to display the list of requested Types in the db such as Material , Equipment , Consumable , Reusable , Tool , and All. 
Or Implements # (WBS) 5.1.4

## Related PRS (if any):
This frontend PR is related to the [BE](https://github.com/OneCommunityGlobal/HGNRest/pull/695) backend PR.

## Main changes explained:
- Implemented modal component and integrated the page with getTypesByType API

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. Go to http://localhost:3000/bmdashboard/equipment/add
6. Test the Check Types button to see if it displays equipment data accurately. 

## Screenshots or videos of changes:

https://www.loom.com/share/2628f969342f4b1b90c2a3a6f7945eca
